code/modules/atmospherics/machinery/components/fusion/hfr_parts.dm

```diff
@@ -392,3 +392,72 @@
 	-In case of a power shortage, the fusion reaction will CONTINUE but the cooling will STOP<BR><BR>\
 	The writer of the quick guide will not be held responsible for misuses and meltdown caused by the use of the guide, \
 	use more advanced guides to understando how the various gases will act as moderators."
+
+/obj/item/hfr_box
+	name = "HFR box"
+	desc = "If you see this, call the police."
+	icon = 'icons/obj/atmospherics/components/hypertorus.dmi'
+	icon_state = "box"
+
+/obj/item/hfr_box/corner
+	name = "HFR box corner"
+	desc = "Place this as the corner of your 3x3 multiblock fusion reactor"
+	icon_state = "box_corner"
+
+/obj/item/hfr_box/body
+	name = "HFR box body"
+	desc = "Place this as the main part of the fusion reactor around the core box"
```

Comment:
This is the side right? like left right up down?

Reply:
yes, i may should word it better.. any way to do that?

Reply:
Maybe like "Place this on the sides of the core box"? I'm maybe not leaning into the lore enough, it's kinda a tricky thing to get across. Or maybe I'm just bad at english